tools/shoestring/shoestring/internal/Preparer.py

```diff
@@ -163,12 +163,15 @@ def create_subdirectories(self):
 			self.directories.userconfig,
 			self.directories.resources
 		]
+
 		if NodeFeatures.API in self.config.node.features:
```

Comment:
you can combined the if statements
``if NodeFeatures.API in self.config.node.features and self.config.node.api_https:``